prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -22,6 +22,8 @@ namespace Prime.Services
 {
     public class EnrolleeService : BaseService, IEnrolleeService
     {
+        private const string PaperGpidPrefix = "NOBCSC";
```

Comment:
Same with front end this is now used in what looks to be three places. Is it possible to have this defined as a constant somewhere and import from one location?